src/pages/hire-developers/four-steps-process.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import Container from "../../components/UiComponent/Container";
+import CountNum from '../../../static/arrow-line.svg'
+import { FourStepsProcessWrapper } from "./style";
+
+const FourStepsProcess = () => {
+  const FourStepsProcessData = [
+    {
+      img: CountNum,
```

Comment:
you can directly access image from static folder like "/arrow-line.svg" no need to import and add this json in constant folder and import here